### PR TITLE
L2 473 filter displaying proposals that by all visible signs should not be shown

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Proposals with `votingPower===0` are visible after enabling the filter (Hide open...) but users was not able to vote using `votingPower===0 neurons`. This PR enables voting with `votingPower===0` neurons. (related nns-dapp [PR](https://github.com/dfinity/nns-dapp/pull/735))

# Changes

- removed `=== 0` filter

# Tests

- updated related test

### CODE REVIEW

File: src/utils/neurons.utils.spec.ts [hunk 1]
```diff
@@ -132,7 +132,7 @@ describe("neurons-utils", () => {
     expect(votable.length).toEqual(1);
   });
 
-  it("should have votable neurons only with voting power", () => {
+  it("should have votable neurons also with voting power", () => {
```
Comment (bitdivine) [inline src/utils/neurons.utils.spec.ts:135]:
```suggestion
  it("should have votable neurons regardless of voting power", () => {
```

File: src/utils/neurons.utils.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ const voteForProposal = ({
   proposalId: ProposalId | undefined;
 }): Vote | undefined => {
   if (!proposalId) {
+    console.error("VFP0", recentBallots);
```
Comment (bitdivine) [inline src/utils/neurons.utils.ts:18]: I assume this is intentional for debugging user problems?  I'd be curious how this could be called without a proposal ID but that is outside the scope of this PR.

**Review by bitdivine — APPROVED** [review]

File: src/utils/neurons.utils.ts (hunk 2 above)
Comment (mstrasinskis) [reply src/utils/neurons.utils.ts]: thanks, removed

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dfinity/nns",
-  "version": "0.4.0",
+  "version": "0.4.1",
```
Comment (peterpeterparker) [inline package.json:3]: we actually did not release `v0.4.0` to npm.

should you revert to 0.3.0 including reverting the changelog?

**Comment by peterpeterparker** [comment]
> (with extra nns-js https://github.com/dfinity/nns-dapp/pull/735)

You might want to update the motivation?

File: package.json (hunk 3 above)
Comment (bitdivine) [reply package.json:3]: Has there been a breaking change since 0.3.0?  I assume that if the version was bumped from 0.3 to 0.4 there was a breaking change, no?

Comment (mstrasinskis) [reply package.json:3]: Your are right, the changelog definitely needs to be updated. Would you prefer to revert the function name `notVotedNeurons` with version update 0.3.0 OR we leave the function name `votableNeurons` without version update?

Comment (peterpeterparker) [reply package.json:3]: I good point the renaming is a breaking change. I actually like the new name.

1. Let's keep `v0.4.0` - revert the version bump in this PR

2. CHANGELOG is alright - nothing to do

3. Let's release a version next week to make things clean (I'll go through the changes to see if I need to add something to the changelog)

Works for you?

Comment (mstrasinskis) [reply package.json]: The version commit was reverted back to 0.4.0. Thank your 👍
